src/views/Home/Home.vue

```diff
@@ -10,7 +10,7 @@
       <holiday-card />
       <schedule-card />
       <!-- TODO: uncomment following when lunch scraper is fixed -->
-      <!-- <lunch-card /> -->
+      <lunch-card />
```

Comment:
I added it back 👍🏻

Reply:
sorry my bad, was looking at the deploy preview (https://deploy-preview-80--shs.netlify.app/) and noticed the lunch card wasn't there, but didn't see that there's no school today 🤦‍♂️, looks good to merge!